cl_dll/hud.cpp

```diff
@@ -524,6 +524,8 @@ void CHud :: Init( void )
 	m_pCvarViewheightMode = CVAR_CREATE("cl_viewheight_mode", "0", FCVAR_ARCHIVE);
 	m_pCvarHideCorpses = CVAR_CREATE("cl_hidecorpses", "0", FCVAR_ARCHIVE);
 	m_pCvarColor = CVAR_CREATE( "hud_color", "", FCVAR_ARCHIVE );
+	m_pCvarHudAlpha = CVAR_CREATE( "hud_alpha", "auto", FCVAR_ARCHIVE );
```

Comment:
Instead of `auto` let's just make the do-nothing value an empty string like `hud_color`.

Reply:
Like this? https://github.com/YaLTeR/OpenAG/pull/126/commits/e82ae2890b72bb1dadbbc45cfcf1bd865efdecc0